crypto/ocsp/ocsp_test.cc

```diff
@@ -769,6 +897,53 @@ TEST_P(OCSPRequestTest, OCSPRequestParse) {
   }
 }
 
+TEST_P(OCSPRequestTest, OCSPRequestSign) {
+  const OCSPReqTestVector &t = GetParam();
+
+  std::string data = GetTestData(std::string("crypto/ocsp/test/aws/" +
+                              std::string(t.ocsp_request) + ".der").c_str());
+  std::vector<uint8_t> ocsp_request_data(data.begin(), data.end());
+  bssl::UniquePtr<OCSP_REQUEST> ocspRequest =
+      LoadOCSP_REQUEST(ocsp_request_data);
+
+  bssl::UniquePtr<X509> server_cert(CertFromPEM(
+      GetTestData(std::string("crypto/ocsp/test/aws/" +
+                        std::string(t.signer_cert) +".pem").c_str()).c_str()));
+  ASSERT_TRUE(server_cert);
+  bssl::UniquePtr<STACK_OF(X509)> additional_cert(CertChainFromPEM(
+      GetTestData(std::string("crypto/ocsp/test/aws/ca_cert.pem").c_str())
+          .c_str()));
+  ASSERT_TRUE(additional_cert);
+
+  if (t.expected_parse_status == OCSP_REQUEST_PARSE_SUCCESS) {
```

Comment:
Should this test the not success path?

Reply:
We're already testing this in `TEST_P(OCSPRequestTest, OCSPRequestParse)`(L730-L768) , so I opted not to make parse testing redundant.